### PR TITLE
add `ctrl+arrow up`/`ctrl+arrow down` keybinding to go first/end row

### PR DESCRIPTION
**Pull request by wugeer**
This PR simply adds support for Home/End keys for moving to first/last row.

### CODE REVIEW

**Comment by jacek-kurlit** [comment]
Hi!
Thank you for PR
The problem is that Home and End are already taken by tui-textarea create that handles input
https://docs.rs/tui-textarea/latest/tui_textarea/#minimal-usage
Currently pressing Home and End will move caret at the beginning and end of search input which may be more intuitive that jumping at top and last row in processes table
If you find this KyeMap useful you may think about other key combination like Ctrl + g/G

**Comment by wugeer** [comment]
> Hi! Thank you for PR The problem is that Home and End are already taken by tui-textarea create that handles input https://docs.rs/tui-textarea/latest/tui_textarea/#minimal-usage Currently pressing Home and End will move caret at the beginning and end of search input which may be more intuitive that jumping at top and last row in processes table If you find this KyeMap useful you may think about other key combination like Ctrl + g/G

Good idea! But I prefer `Ctrl + Up/Down` keys to move to the beginning/end, while pressing the Up/Down keys alone represents moving one line. :)

File: src/tui/rendering.rs [hunk 1]
```diff
@@ -66,6 +66,19 @@ impl Tui {
         }
     }
 
+    pub fn select_first_row(&mut self) {
+        self.select_row_by_index(Some(0));
```
Comment (jacek-kurlit) [inline src/tui/rendering.rs:70]: How this will behave when there is no processes displayed and you move to the first row?

File: src/tui.rs [hunk 2]
```diff
@@ -108,6 +108,12 @@ fn run_app<B: Backend>(terminal: &mut Terminal<B>, mut app: App) -> io::Result<(
                 use KeyCode::*;
                 match key.code {
                     Esc => return Ok(()),
+                    Up if key.modifiers.contains(KeyModifiers::CONTROL) => {
```
Comment (jacek-kurlit) [inline src/tui.rs:111]: Please update readme key mappings

**Review by jacek-kurlit — APPROVED** [review]

**Pull request merged**